src/App.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div id="app">
+    <h1 class="title">  Addons catalog </h1>
+    <table v-if="addons.length > 0">
+      <thead>
+        <tr>
+          <th>
+            Name
+          </th>
+          <th>
+            Version
+          </th>
+          <th>
+            Unstable
+          </th>
+          <th>
+            Compatibility
+          </th>
+          <th>
+            Download
+          </th>
+        </tr> 
+      </thead>
+      <tbody>
+        <tr v-for="addon in addons">
+          <td>
+            {{addon.name}}
+          </td> 
+          <td>
+            {{addon.version}}
+          </td> 
+          <td>
+            {{addon.unstable}}
+          </td>
+          <td>
+            {{addon.compatibility}}
+          </td>
+          <td> 
+            <a :href="addon.downloadUrl" ><i class="uiIconDownload"></i></a>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+export default {
+  name: 'App',
+   created() {
+    this.fetchData();	
+  },
+  data() {
+    return {
+      addons: []
+    }
+  },
+  methods: {
+    fetchData() {
+      axios.get('http://storage.exoplatform.org/public/Addons/list.json').then(response => {
+        this.addons = response.data;
+      });
```

Comment:
We should add:
* a loading image while the catalog is being fetched (it can take seconds with a bad network)
* an error message when the catalog cannot be fetched (you can use the error message pattern from the UX guidelines)